inference-engine/tests/ngraph_functions/src/low_precision_transformations/round_function.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/low_precision_transformations/round_function.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "ngraph_functions/low_precision_transformations/common/builders.hpp"
+
+using namespace ngraph::pass::low_precision;
+
+namespace ngraph {
+namespace builder {
+namespace subgraph {
+    std::shared_ptr<ngraph::Function> RoundWithToleranceFunction::getOriginal(
```

Comment:
minor comment. I think will be more clear to add LPT namespace for this util

Reply:
will be resolved later: issue #43757